docs/source/gbd2017_models/risk_attributable_causes/iron_deficiency/index.rst

```diff
@@ -584,23 +573,33 @@ within the population hemoglobin concentration distribution) is a limitation of
 our modeling strategy in that it assumes that this remains constant over time
 and age groups.
 
-Because we are not modeling individual causes of anemia (and their associated
-mortality), we assume that all simulants have the same mortality rate regardless
-of their hemoglobin value, when in reality, those with lower hemoglobin values
-will have higher mortality rates than those with higher hemoglobin values.
-Notably, deaths due to causes with iron responsive anemia sequelae account for
-approximately 1.1% of deaths in the first five years of life globally.
+We assume that all individuals who are not anemic are iron responsive. This 
+assumption is based on two additional assumptions, including a) asymptomatic 
+individuals with non-iron-responsive causes will remain asymptomatic, and b) 
+the prevalence of individuals with non-iron-responsive causes without anemia 
+is neglible (see a notebook investigating the prevalence of these groups `here <https://github.com/ihmeuw/vivarium_data_analysis/blob/master/pre_processing/iron_deficiency_risk_factor/Iron%20responsiveness%20among%20the%20non-anemic.ipynb>`_). 
```

Comment:
I'm a bit confused about what this is saying, but maybe I just need to think about it some more...